go/cmd/dolt/commands/diff.go

```diff
@@ -691,3 +750,36 @@ func writeDiffResults(
 		}
 	}
 }
+
+func getModifiedCols(
```

Comment:
(minor) some simple godocs for this function would be nice, just to help future code readers understand the intent and any behavior quirks quickly. Explaining the `unionSch` and `diffQuerySch` params would probably be helpful for future readers, too.

Reply:
Beautiful docs! Thank you @tanyongzhi!